spec/php-spec.coffee

```diff
@@ -1863,6 +1863,28 @@ describe 'PHP grammar', ->
     expect(tokens[4]).toEqual value: '\'', scopes: ['source.php', 'string.quoted.single.php', 'punctuation.definition.string.end.php']
     expect(tokens[5]).toEqual value: ';', scopes: ['source.php', 'punctuation.terminator.expression.php']
 
+  it 'should tokenize heredoc in a function call correctly', ->
+    lines = grammar.tokenizeLines  '''
+    foo(
+    <<<HEREDOC
```

Comment:
Indent the lines inside foo() by 2 or 4 spaces, right now it looks like this:

```php
foo(
<<<HEREDOC
This is just a cool test.
HEREDOC,
$bar
);
```

but we want

```php
foo(
  <<<HEREDOC
  This is just a cool test.
  HEREDOC,
  $bar
);
```

Reply:
@Ingramz Looks like failure ... But without indention checks were good.

Reply:
You'll need to update expect()... lines too, because now there is one extra token in front of every line.